src/Editor/GUI/Editors/EntityEditor.hpp

```diff
@@ -74,6 +76,7 @@ namespace GUI {
             
             // Editors
             void AnimationEditor(Component::Animation* animation);
+            // @todo: remove when physics component is gone
```

Comment:
If you use ///, the todo will appear in a todo-list in the generated doxygen documentation.